tabs/pid_tuning.js

```diff
@@ -442,7 +442,7 @@ TABS.pid_tuning.initialize = function (callback) {
 
         updatePidDisplay();
 
-        showAllButton.on('click', function(){
+        $('#showAllPids').on('click', function(){
```

Comment:
Actually this is not was broke the button, it was the unwrapping of functions into the outer closure that broke it. Unfortunately, your fix, while making the button work again, does not fix the changing of the text on the button. Fixed in #120.